activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/SignalAuditProducerIT.java

```diff
@@ -81,6 +84,18 @@ public class SignalAuditProducerIT {
     @Autowired
     private ProcessDefinitionRestTemplate processDefinitionRestTemplate;
 
+    @AfterEach
+    public void cleanUp() {
+        List<String> processInstanceIds = runtimeService.createProcessInstanceQuery()
+            .list()
+            .stream()
+            .map(ProcessInstance::getProcessInstanceId)
+            .collect(Collectors.toList());
+
+        processInstanceIds.forEach(processInstanceId -> runtimeService
```

Comment:
There should be only one process instance. Can you assert that only one process instance?

Reply:
I think we already assert that during the test. I think the cleanUp should not assert anything

Reply:
https://github.com/Activiti/activiti-cloud/pull/921/files#diff-2e71ce483242aa1ac6962d147a0ec4d94f1401ec5840e718fed9af05730119c0R225

Reply:
but what about the other unit test method?

Reply:
This should cover the case for the other test thhttps://github.com/Activiti/activiti-cloud/blob/b5b9ea66176c9f3af2b89b9d0bac81a49ff0471e/activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/SignalAuditProducerIT.java#L146

Reply:
Ok :) However you can do a singleResult in the cleanUp too.